jetty-proxy/src/main/java/org/eclipse/jetty/proxy/AbstractProxyServlet.java

```diff
@@ -639,16 +643,21 @@ protected void onProxyResponseFailure(HttpServletRequest clientRequest, HttpServ
         if (_log.isDebugEnabled())
             _log.debug(getRequestId(clientRequest) + " proxying failed", failure);
 
-        int status = failure instanceof TimeoutException ?
-            HttpStatus.GATEWAY_TIMEOUT_504 :
-                HttpStatus.BAD_GATEWAY_502;
+        int status = proxyResponseStatus(failure);
         int serverStatus = serverResponse == null ? status : serverResponse.getStatus();
         if (expects100Continue(clientRequest) && serverStatus >= HttpStatus.OK_200)
             status = serverStatus;
         sendProxyResponseError(clientRequest, proxyResponse, status);
         
     }
 
+	protected int proxyResponseStatus(Throwable failure) 
+	{
+		return failure instanceof TimeoutException ? 
```

Comment:
Too much indentation for return. Use 4 spaces, no tabs.
Broken line gets indented 8 spaces.